### DISCUSSION TITLE
What does the Lotus Daemon do? What kind of machine is supposed to run it?

### DISCUSSION THREAD

**Discussion by Geo25rey** [filecoin-project/lotus#4499] [Storage Provider] [answered]
I've tested the Lotus Daemon on both my laptop (with an NVMe SSD) and my desktop (with a standard 7200rpm HDD). On my desktop, the Lotus Daemon could never finish syncing; I'm assuming because the HDD is just too slow to keep up. On my laptop, syncing keeps up, but the Lotus Daemon acquires half my RAM (~8GB) and periodically consumes 99% of my CPU's time for anywhere from 1-10 minutes during syncing. This leads into my question.

What is the Lotus Daemon supposed to run on? Is it for $2000 server with many TB of NVMe storage? Is it supposed to be for any one who wants to share files on IPFS?

Perhaps to make the Lotus Daemon more accessible to lower end systems, certain "modules" can be enabled or disabled with the discression of the user. For examples, those who just want to share files on IPFS don't need to know when anyone requests a deal. I could be way off in my suggestion, since Lotus seems to be kind of a black box to me. All I understand is that the Lotus Daemon syncs with the Filecoin mainnet and you can request a deal with a miner to store your data.

Am I thinking about the Lotus Daemon correctly? What am I missing? How does the sync process actually work?

**Comment by jennijuju** [comment, marked as answer]
First of all, welcome! 

You can find the lotus minimal reqiurement [here](https://docs.filecoin.io/get-started/lotus/installation/#minimal-requirements), this essentially your node and what you use to sync with the network, so yes, you are thinking it correctly. The mainnet full chain size is huge and is increasing 9GiB/day. So it is safe to say it's nearly impossible to sync from scratch. Thats why id suggest you to [sync from a snapshot](https://docs.filecoin.io/get-started/lotus/chain/#syncing-from-a-trusted-state-snapshot-mainnet). and with the minimal one on my mac it takes about 2 hours excluding downloading the params.
